pyvista/core/pointset.py

```diff
@@ -27,6 +29,10 @@
 
 log = logging.getLogger(__name__)
 log.setLevel('CRITICAL')
+DEFAULT_INPLACE_WARNING = 'You did not specify a value for `inplace` and the ' \
+    'defualt for value will be changing to `False` in future versions for ' \
+    'point-based meshes (e.g., `PolyData`). Please make sure you are not ' \
+    'assuming this to be an inplace operation.'
```

Comment:
Python docs suggest using string literal concatenation: https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation

Also a typo in default in case you don't like this suggestion

```suggestion
DEFAULT_INPLACE_WARNING = (
    'You did not specify a value for `inplace` and the '
    'default for value will be changing to `False` in future versions for '
    'point-based meshes (e.g., `PolyData`). Please make sure you are not '
    'assuming this to be an inplace operation.'
)
```